host/allwinner/gpio.go

```diff
@@ -216,6 +219,17 @@ func (p *Pin) Out(l gpio.Level) error {
 		}
 		p.usingEdge = false
 	}
+	p.FastOut(l)
+	p.setFunction(out)
+	return nil
+}
+
+// FastOut sets a pin output level with Absolutely No error checking.
```

Comment:
same bas before, I'm curious what could go wrong, what kind of errors and what could be the consequences.

Reply:
In practice:
- If the driver didn't load, it will cause a panic
- otherwise, nothing happens

Reply:
I'd say that's something we'd want to let the users know because it will help them decide to use this faster approach or not

Reply:
I disagree. The documentation tells them to use Out() first, if they don't, there is no guarantee. If they use Out() first, the behavior is defined. I don't think there is any more need to document the lack of guarantees. In particular I don't want to document the failing modes as I want to retain the right to change them.